### PR TITLE
3.1.11

### PR DESCRIPTION
**Pull request by WilRyan**
Testing on php 72, fixed a whole bunch of WC errors

### CODE REVIEW

File: MdsSupportingClasses/MdsColliveryService.php [hunk 1]
```diff
@@ -555,16 +554,16 @@ public function automatedAddCollivery($order_id, $processing = false)
...
-                        'email' => str_replace(' ', '', $order->shipping_email),
-                        'custom_id' => $order->user_id,
+                        'company_name' => ($order->get_shipping_company() != '') ? $order->get_shipping_company() : 'Private',
+                        'building' => $order->get_shipping_address_2(),
+                        'street' => $order->get_shipping_address_1(),
+                        'location_type' => $order->get_meta('_shipping_location_type'),
+                        'suburb' => $order->get_meta('_shipping_suburb'),
+                        'town' => $order->get_shipping_city(),
+                        'full_name' => $order->get_shipping_first_name() . ' ' . $order->get_shipping_last_name(),
+                        'cellphone' => preg_replace('/[^0-9]/', '', $order->get_billing_phone()),
```
Comment (brycelarge) [inline MdsSupportingClasses/MdsColliveryService.php]: str_replace can become trim($order->get_billing_email()) or removed

File: collivery.php [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 define('_MDS_DIR_', __DIR__);
-define('MDS_VERSION', '3.1.8');
+define('MDS_VERSION', '3.1.11');
 include 'autoload.php';
 
 /*
  * Plugin Name: MDS Collivery
  * Plugin URI: https://collivery.net/integration/woocommerce
  * Description: Plugin to add support for MDS Collivery in WooCommerce.
```
Comment (brycelarge) [inline collivery.php]: What about the docs, the supported version is also listed there?

File: collivery.php [hunk 3]
```diff
@@ -1,18 +1,18 @@
 <?php
 
 define('_MDS_DIR_', __DIR__);
```
Comment (brycelarge) [inline collivery.php]: Just as a side note, When Woocommerce jumps to version 4 this version should also jump

File: MdsSupportingClasses/MdsColliveryService.php (hunk 1 above)
Comment (MarnuLombard) [reply MdsSupportingClasses/MdsColliveryService.php]: Why not replace all non digits

File: collivery.php (hunk 2 above)
Comment (WilRyan) [reply collivery.php]: Thanks, will update there as well

File: collivery.php (hunk 3 above)
Comment (WilRyan) [reply collivery.php]: Thanks

File: MdsSupportingClasses/MdsColliveryService.php (hunk 1 above)
Comment (WilRyan) [reply MdsSupportingClasses/MdsColliveryService.php]: I just changed the code referencing the data, was not gonna touch what was there.

**Pull request merged**